docker/docker-entrypoint-headless.sh

```diff
@@ -6,34 +6,36 @@ set -e
 echo "${MJKEY}" > /root/.mujoco/mjkey.txt
 
 # Setup dummy X server display
-display_num=0
+# Socket for display :0 may already be in use if the container is connected
+# to the network of the host, that's why we use 1.
```

Comment:
in that case you should start with a very high display number (e.g. 100) to avoid collisions with similar reasons.